asset_stream_manager/asset_stream_config.cc

```diff
@@ -195,5 +250,18 @@ std::string AssetStreamConfig::GetFlagReadErrors() {
                                  error_str.empty() ? "" : "\n", flag, error);
   return error_str;
 }
+std::function<void(const std::string&)> AssetStreamConfig::JedecParser(
+    const char* flag_name, uint64_t* bytes) {
+  return [flag_name, bytes,
+          error = &jedec_parse_error_](const std::string& value) {
+    JedecSize size;
+    if (AbslParseFlag(value, &size, error)) {
```

Comment:
This CL removes dependencies on absl libraries for flags and their parsing.. I am wondering how this works now..